cri/Makefile

```diff
@@ -37,9 +37,28 @@ test:
 	KUBECONFIG=/etc/kubernetes/admin.conf kn service apply helloworldserial -f ./../configs/knative_workloads/helloworldSerial.yaml
 	KUBECONFIG=/etc/kubernetes/admin.conf kn service apply pyaes -f ./../configs/knative_workloads/pyaes.yaml
 	sleep 1m
-
+	go clean -testcache
```

Comment:
I doubt that it's necessary, why?

Reply:
When you run both tests (cri-firecracker, cri-gvisor) after another, the tests of the second run will immediately return the result of the first test, because nothing has changed from the code, and the tests are exactly the same.

Reply:
makes sense ok